.github/workflows/ci.yml

```diff
@@ -788,8 +788,51 @@ jobs:
             docker-file: camunda.Dockerfile
             owner: "Data Layer"
             module: "Schema Manager"
-          - group: qa-integration
-            name: "Zeebe QA"
+          - group: zeebe-engine-integration-legacy
+            name: "Zeebe Legacy QA (just testing)"
+            maven-modules: "zeebe/qa/integration-tests"
+            maven-build-threads: 1
+            maven-test-fork-count: 10
+            runs-on: gcp-perf-core-16-default
+            docker-repository: localhost:5000/camunda/zeebe
+            docker-file: Dockerfile
+            owner: "Core Features"
+            module: "Zeebe"
+          - group: zeebe-engine-integration
+            name: "Zeebe Engine QA"
+            maven-modules: "zeebe/qa/integration-tests"
+            maven-build-threads: 1
+            maven-test-fork-count: 10
+            runs-on: gcp-perf-core-16-default
+            docker-repository: localhost:5000/camunda/zeebe
+            docker-file: Dockerfile
```

Comment:
🙃 I checked superficially and it looks like none of the engine integration tests require us to build a docker image. It'd be nice to skip that.